realm/src/androidTest/java/io/realm/internal/JNIQueryTest.java

```diff
@@ -315,19 +315,19 @@ public void testNegativeColumnIndexContains() {
         TableQuery query = table.where();
 
         // String
-        try { query.contains(-1, "hey").findAll(); fail("-1 column index"); } catch (ArrayIndexOutOfBoundsException e) {}
-        try { query.contains(-1, "hey").findAll(); fail("-10 column index"); } catch (ArrayIndexOutOfBoundsException e) {}
-        try { query.contains(-1, "hey").findAll(); fail("-100 column index"); } catch (ArrayIndexOutOfBoundsException e) {}
+        try { query.contains(new long[]{-1}, "hey").findAll(); fail("-1 column index"); } catch (ArrayIndexOutOfBoundsException e) {}
```

Comment:
This is testing the same thing 3 times (previous copy/paste error it seams).
But testing with -1, -10, -100 are bad test values anyway. It should be edge values instead.